shared/src/main/java/uk/gov/di/authentication/entity/TICFCRIRequest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.authentication.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.util.List;
+
+public record TICFCRIRequest(
+        @Expose @Required @SerializedName("sub") String internalPairwiseId,
+        @Expose @Required @SerializedName("vtr") List<String> vtr,
+        @Expose @Required @SerializedName("govuk_signin_journey_id") String journeyId,
```

Comment:
Consider calling the variable govUkSigninJourneyId then the @SerializedName annotation can be dropped.